tests/integration/helpers.py

```diff
@@ -114,7 +114,7 @@ async def run_action(
                 continue
 
             ping = subprocess.call(
-                f"ping -c 1 {unit.ip}".split(),
+                f"nc -zv {unit.ip} 9200".split(),
```

Comment:
This is not the goal of this ping. This will bias integration tests and set them in an ideal scenario. 
We are not attempting to prevent actions from being run until after the opensearch instance is up, what we want instead is to run actions after the **juju unit** is up.

Reply:
Actually, we can target port 22 or 17070, they must be open since the start